README.md

```diff
@@ -1,84 +1,96 @@
-# pikdum's steam deck tools
+# Pikdum's Steam Deck Tools
 
 > **Notice:** Recently swapped Vortex install method from STL to [pikdum/vortex-linux](https://github.com/pikdum/vortex-linux).  
 > This is a breaking change, so you'll need to uninstall + reinstall.  
 > It's also completely new, so there might be bugs.  
 
-## what is this
+## What is this
 
-a collection of steam deck tools and scripts to help automate some things, starting with installing vortex
+A Collection of tools and scripts for linux/steam-deck. 
 
-## install
+Current Tools:
+- Vortex Installer
+- Vortex Fixes for linux/steamdeck
 
-1. right click and save as [this install.desktop link](https://raw.githubusercontent.com/pikdum/steam-deck/master/install.desktop)
-2. go to downloads folder and double click to run it
+## Install
 
-## vortex
+1. Right click and save as [this install.desktop link](https://github.com/SirStig/Steam-deck-tests/releases/download/Testing/Install-pikdums-tools.desktop)
```

Comment:
**issue:** This should reference this repo.

Maybe there's some clever way to do a relative link?